src/ports.js

```diff
@@ -31,6 +31,24 @@
             this.ports.push(port);
         },
 
+        storeLayoutPosition: function (id, position) {
```

Comment:
shouldn't this be called `setLayoutPosition()`? I guess we have "getters" and "setters", not "getters" and "storers".